src/v/cloud_storage/tests/BUILD

```diff
@@ -194,3 +199,20 @@ redpanda_cc_btest(
         "@seastar//:testing",
     ],
 )
+
+redpanda_cc_gtest(
+    name = "remote_path_provider_test",
+    timeout = "short",
+    srcs = [
+        "remote_path_provider_test.cc",
```

Comment:
i think this was added in https://github.com/redpanda-data/redpanda/pull/23695 did you mean to add something else?

Reply:
🤦‍♂️ apologies for this. No idea how i cherry-picked it from another PR to get to such a state. 🤯